src/Lecture1.hs

```diff
@@ -77,7 +80,11 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Int -> Int -> Int -> Int
+minmax x y z
+  | x > y && x < z || x < y && x > z = abs (y - z)
+  | y < x && y > z || y > x && y < z = abs (x - z)
+  | otherwise = abs (x - y)
```

Comment:
That's a nice solution! Alternatively, you can use standard functions `min` and `max` 👍🏻